introduction/intro_to_js.md

```diff
@@ -1,138 +0,0 @@
-# Intro to JavaScript world
```

Comment:
moved to `lesson-0/*`, looks like I changed it so much git didn't recognize the move :(